README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">  
-  <a href="https://formidable.com/open-source/" target="_blank">
-    <img alt="FigLog — Formidable, We build the modern web" src="./example.png" />
+  <a href="https://commerce.nearform.com/open-source/" target="_blank">
+    <img alt="FigLog Evolution Tracker by Formidable now Nearform - We build beautifully designed, solidly engineered, performant digital experiences for humans, just like you." src="./example.png" />
   </a>
 
   <h3>
```

Comment:
Can We update "Design Evolution Widget for Figma to"

FigLog Evolution Tracker Widget for Figma